packages/aws-cdk/lib/api/bootstrap/bootstrap-environment.ts

```diff
@@ -139,16 +139,20 @@ export class Bootstrapper {
     *    - the name indicating the custom permissions boundary to be used
     * Re-bootstrapping will NOT be blocked by either tightening or relaxing the permissions' boundary.
     */
-    const currentPermissionsBoundary = current.parameters.InputPermissionsBoundary;
+    const currentPermissionsBoundary: string | undefined = current.parameters.InputPermissionsBoundary || undefined;
```

Comment:
This doesn't change anything, right? The old type used to be `"<something-useful>" | "" | undefined`, and it still is?

Reply:
Ehrmagerd now I get it, it's a load-bearing `||` ! 

A comment please, in my head I interpreted it as `??`.